mabwiser/base_mab.py

```diff
@@ -64,11 +68,11 @@ class BaseMAB(metaclass=abc.ABCMeta):
         Default value is None. In this case the default backend selected by joblib will be used.
     arm_to_expectation: Dict[Arm, float]
         The dictionary of arms (keys) to their expected rewards (values).
-    cold_arm_to_warm_arm: Dict[Arm, Arm]:
-        Mapping indicating what arm was used to warm-start cold arms.
-    trained_arms: List[Arm]
-        List of trained arms.
-        Arms for which at least one decision has been observed are deemed trained.
+    arm_to_status: Dict[Arm, dict]
+        The dictionary of arms (keys) to their status (values), where the status consists of
```

Comment:
so what happens when an arm is warm_started and then trained on top?

Reply:
When arm is partial fit `is_trained` will change to True, but we keep `is_warm` as True together with info in `warm_started_by` in order to track that the arm was originally warm started.

Reply:
My bad.. I quickly skimmed and missed that "status" is actually a dictionary itself.. Thought it was singleton. keeping the history makes total sense. May be a quick pydoc comment on exactly what you have above for that warm+fine-tuned case.